services/fischcard.service.ts

```diff
@@ -55,3 +51,18 @@ export const updateCard = async (
 
   return updatedCard;
 };
+
+export const deleteCardWhenTimePassed = async (
+  card: CardPayload,
+  minutes: number
+): Promise<CardPayload> => {
+  const currentTime = new Date();
+  const cardCreationTime = new Date(`${card.date}`);
+  const timeDifferenceInMinutes =
+    (currentTime.getTime() - cardCreationTime.getTime()) / 60000;
```

Comment:
**nitpick:** I would suggest to throw Error here when `timeDifferenceInMinutes >= minutes`

Reply:
I didn't understand the task